packages/astro/src/@types/astro.ts

```diff
@@ -881,12 +881,14 @@ export interface AstroUserConfig {
 		/**
 		 * @docs
 		 * @name build.assetsPrefix
-		 * @type {string}
+		 * @type {string | Record<string, string>}
 		 * @default `undefined`
 		 * @version 2.2.0
 		 * @description
 		 * Specifies the prefix for Astro-generated asset links. This can be used if assets are served from a different domain than the current site.
 		 *
+		 * If this value is a `string`,
+		 *
 		 * For example, if this is set to `https://cdn.example.com`, assets will be fetched from `https://cdn.example.com/_astro/...` (regardless of the `base` option).
 		 * You would need to upload the files in `./dist/_astro/` to `https://cdn.example.com/_astro/` to serve the assets.
 		 * The process varies depending on how the third-party domain is hosted.
```

Comment:
```suggestion
```
(Also, I can't suggest editing the line below this, so in fact what I would prefer is that the two existing lines are removed. Then, only one line after the code sample with a short note about the "this might vary". Something like:
```
* To fetch all assets uploaded to the same domain (e.g. `https://cdn.example.com/_astro/...`), set `assetsPrefix` to the root domain as a string (regardless of your `base` configuration):
[code sample]
* (Note: This process may vary depending on how the third-party domain is hosted.)
```